### PR TITLE
Common header and footer

### PR DESCRIPTION
**Pull request by Svetlana7899**
Common header and footer

### CODE REVIEW

**Comment by ivan-perl** [comment]
I like overall idea, but I have few comments. Will post them as a separate messages to make it possible to discuss and comment separately.

File: cosm-lab.site/ru/footer.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                    </div>
+                    <div class="footer-center">
+                        <span>
+                            designed by
+                            <a href="https://www.behance.net/svalion" target="_blank"
+                                style="text-decoration:none; color: lightgray;">
+                                @svalion
+                            </a>
+                            <br />
+                            COSM lab &copy; 2019-2021. All rights reserved
```
Comment (ivan-perl) [inline cosm-lab.site/ru/footer.html]: Footer always indicates 2021, c-mon ) we are still alive, please make it pointing always to the current year

File: cosm-lab.site/ru/footer.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<div id="content-wrap">
```
Comment (ivan-perl) [inline cosm-lab.site/ru/footer.html:1]: Both for header and footer, can we extract them also in en version, not only in ru? I want to keep both versions as alike as possible

File: cosm-lab.site/ru/header.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<div id="content-wrap">
```
Comment (ivan-perl) [inline cosm-lab.site/ru/header.html:1]: Currently, I see that header looks different on main page and, for example, on projects page. We need to make them look alike, so when user switching from one page to another header looks the same.

File: cosm-lab.site/ru/header.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    <header class="header-without-wallpaper">
```
Comment (ivan-perl) [inline cosm-lab.site/ru/header.html:2]: Currently we extracted header and footer for projects and topics pages, can we also do this for index.html? So all pages in each locale will have the same header and footer

File: cosm-lab.site/ru/footer.html (hunk 1 above)
Comment (Svetlana7899) [reply cosm-lab.site/ru/footer.html]: Fixed, thanks.

File: cosm-lab.site/ru/footer.html (hunk 2 above)
Comment (Svetlana7899) [reply cosm-lab.site/ru/footer.html:1]: Fixed, thanks.

File: cosm-lab.site/ru/header.html (hunk 3 above)
Comment (Svetlana7899) [reply cosm-lab.site/ru/header.html:1]: Fixed, thanks.

File: cosm-lab.site/ru/header.html (hunk 4 above)
Comment (Svetlana7899) [reply cosm-lab.site/ru/header.html:2]: Fixed, thanks.

**Review by ivan-perl — APPROVED** [review]

**Pull request merged**